### DISCUSSION TITLE
Not allowing me to change home/away mode.

### DISCUSSION THREAD

**Discussion by pricejordan41** [dgreif/ring#641] [Q&A]
My RING devices are all showing up in HomeKit except the Security Alarm base station. When I originally set this up, it all worked correctly. Somewhere along the line it just disappeared. The plugin is all the way up to date, and Ive even tried uninstalling/reinstalling the plugin and restarting the server. The Base Station does appear in the HOOBS dashboard though. Any idea?? TIA

**Comment by dgreif** [comment]
Are you using a shared account to log into Ring from homebridge?  Sometimes device permissions get messed up in that scenario

**Reply by pricejordan41** [reply]
Not sure what you mean by this.. I have two locations on my account, one is my personal home and the other is my place of employment which does share the cameras with me. Is that what you mean? If so what would be a solution to fix this?

**Reply by pricejordan41** [reply]
Hi.... any idea...?

**Reply by dgreif** [reply]
Can you post your config (minus refreshToken) for the plugin?  Are you referring to the actual Base Station component (only supports volume controls and battery status on the base station), or the Alarm control, which allows you to switch between home/away?

**Reply by pricejordan41** [reply]
I am referring to the alarm control for home, away, etc. I understand that "hiding of the unsupported devices" will hide the base station - but before it still aloud alarm control. 

I will post my config as a soon as I get home.

**Reply by dgreif** [reply]
And you see the Alarm control in HOOBS? Is it possible that it is in a Room in the Home app that you aren't expecting?  Maybe it got removed from favorites?

**Reply by pricejordan41** [reply]
Yes it's in my Hoobs dashboard and works as it should. I've checked every room I can, but I don't know how to like view all of the accessories in HomeKit at once. I even went to the hoobs server, clicked on connected devices and couldn't locate it within HomeKit.

**Reply by pricejordan41** [reply]
Here is my configuration settings. 
![Screen Shot 2021-05-13 at 8 38 32 PM](https://user-images.githubusercontent.com/83792808/118207921-4807db80-b42b-11eb-8413-37ba640bfb66.png)

**Reply by pricejordan41** [reply]
![Screen Shot 2021-05-13 at 8 40 00 PM](https://user-images.githubusercontent.com/83792808/118208026-8a311d00-b42b-11eb-92c3-6b10ba9ac6f6.png)
![Screen Shot 2021-05-13 at 8 40 32 PM](https://user-images.githubusercontent.com/83792808/118208028-8ac9b380-b42b-11eb-86e0-8f905bffbaa5.png)
![Screen Shot 2021-05-13 at 8 40 48 PM](https://user-images.githubusercontent.com/83792808/118208029-8ac9b380-b42b-11eb-98ea-6c26093180f5.png)

**Reply by pricejordan41** [reply]
Any idea..?

**Reply by pricejordan41** [reply]
I'm assuming this is a loss cause?

**Reply by dgreif** [reply]
Sorry @pricejordan41 I haven't had a chance to look into this any further.  If the alarm is showing up in HOOBS, then it _should_ show up in HomeKit.  I'd reach out to HOOBS support and see if they have any ideas.

**Reply by dgreif** [reply]
@pricejordan41 did you ever get this figured out?
